### ISSUE TITLE
Latitude 7490 support?

### ISSUE THREAD

**Issue by Neurrone** [kevinshroff/KSMRD-Modded-Realtek-Audio-Drivers#4]
Hi,

Does this support the Latitude 7490?

**Comment by kevinshroff** [comment]
Go to Device Manager - Sound, video and game controllers, and right click Realtek Audio.
Click properties, then click on tab "Details", and then under the Property drop-down list select Hardware IDs and copy and paste the second line to the GitHub issue, asking me to verify or add support for that device.

**Comment by Neurrone** [comment]
Hardware ID: `HDAUDIO\FUNC_01&VEN_10EC&DEV_0256&SUBSYS_1028081C`

Could you add support for this? Just tried installing it on the 7490, and it doesn't seem to work.

**Comment by Neurrone** [comment]
Bump?

**Comment by kevinshroff** [comment]
Read https://github.com/kevinshroff/KSMRD-Modded-Realtek-Audio-Drivers/issues/9
TLDR: I will not start working on KSMRD again until mid December

**Comment by Neurrone** [comment]
Any idea when you'll have the time to resume development?

**Comment by moriel5** [comment]
@kevinshroff already has.
Check the experimental branch.

By the way, in general it is bad manners to ask for an ETA.

**Comment by Neurrone** [comment]
@moriel5 thanks. Noted, didn't mean it like that.

**Comment by moriel5** [comment]
@Neurrone don't worry, I did not mean anything vad about you, it's a general rule, which I had first learned over at XDA Developers, however it is true for any development source.
